Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 91.5.6-snapshot
```

Comment:
Why not using 0.0.1-snapshot?

Reply:
sorry this is coming from the last commit which is a test one. gonna revert that

Reply:
reverted